src/panel/direction/DirectionInput.jsx

```diff
@@ -15,34 +15,33 @@ class DirectionInput extends React.Component {
   }
 
   componentDidMount() {
-    this.suggest = new Suggest({
-      tagSelector: `#itinerary_input_${this.props.pointType}`,
-      onSelect: this.selectItem,
-      prefixes: [ NavigatorGeolocalisationPoi.getInstance() ],
-      menuClass: 'direction_suggestions',
-    });
+    if (this.props.claimFocus) {
+      this.focus();
+    }
   }
 
-  componentWillUnmount() {
-    if (this.suggest) {
-      this.suggest.destroy();
+  componentDidUpdate() {
+    if (this.props.claimFocus) {
```

Comment:
We changed how we managed this in recent merges, don't wait too long before rebase :)